HelpfulItem.h

```diff
@@ -10,6 +10,5 @@
 struct HelpfulItem : Item
```

Comment:
`HelpfulItem` does not need to include all of these source files.
you learned about "Don't Include what you Don't Use" in project 5.4
- [ ] revise